db/migrate/20250626150847_remove_not_null_from_trs_teacher_names.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveNotNullFromTRSTeacherNames < ActiveRecord::Migration[8.0]
+  # rubocop:disable Rails/BulkChangeTable
+  def up
+    remove_column :teachers, :search, :tsvector, type: :tsvector, as: "to_tsvector('unaccented', #{tsvector_columns})", stored: true
+    change_column :teachers, :trs_first_name, :string, null: true
+    change_column :teachers, :trs_last_name, :string, null: true
+    add_column :teachers, :search, :tsvector, type: :tsvector, as: "to_tsvector('unaccented', #{tsvector_columns})", stored: true
+    add_index :teachers, :search, using: :gin
+  end
+
+  def down
+    remove_column :teachers, :search, :tsvector, type: :tsvector, as: "to_tsvector('unaccented', #{tsvector_columns})", stored: true
+    change_column :teachers, :trs_first_name, :string, null: false, limit: 80
```

Comment:
There is no limit on the `up` migration or the schema but the down migration includes `limit: 80`, did you mean to add this here?

Also is `add_index :teachers, :search, using: :gin` missing from `down` too?